File: 268_MissingNumber/solution.py
class Solution(object):
    def missingNumber(self, nums):
        total = ((len(nums) + 1) * len(nums))/2
        Sum = 0
        for i in range(len(nums)):
            Sum += nums[i]
        return total - Sum
        """
        :type nums: List[int]
        :rtype: int
        """
        

File: 268_MissingNumber/solution.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int sum = 0;
        int total = 0;
        for(int i = 0; i < size(nums); i++){
            sum += nums[i];
            total = total + 1 + i;
        }
        return total - sum;
    }
};
